index.bs

```diff
@@ -661,6 +587,65 @@ by using the raw output of a gyroscope
 and waiting for quick rotational movements of the head
 to compensate for drift.
 
+## Default sensor ## {#concepts-default-sensor}
+
+The Generic Sensor API is designed to make the most common use cases straightforward
+while still enabling more complex use cases.
+
+Most of devices deployed today do not carry more than one
+[=device sensor=] corresponding to the same [=sensor type|sensor types=].
+The use cases which require a set of similar [=device sensors=] are rare
+and generally limited to specific [=sensor types=],
+such as proximity sensors.
```

Comment:
not really, accelerometers are better example, especially for 2-in-1 or convertible devices. Maybe change proximity to accelerometer?